components/Layout.js

```diff
@@ -31,7 +31,7 @@ const Layout = ({ links, children, title, description }) => {
   const [secondaryNavbar, setSecondaryNavbar] = useState(false);
 
   useEffect(() => {
-    if (currentPath === "/hackrplay/2022/home") {
+    if (currentPath === "/hackrplay/2022/home" || currentPath === "/hackrplay/2023/home") {
```

Comment:
the 2023 event is not hackrplay. Please check the actual description of the issue.

Reply:
yes got it